integration/tests/cook/test_cli.py

```diff
@@ -295,30 +294,27 @@ def test_retries(self):
         cp, uuids = cli.submit('-- ls -al', 'localhost:99999', '--verbose')
         stderr = cli.decode(cp.stderr)
         self.assertEqual(1, cp.returncode, stderr)
-        self.assertIn('Starting new HTTP connection (1)', stderr)
-        self.assertIn('Starting new HTTP connection (2)', stderr)
-        self.assertIn('Starting new HTTP connection (3)', stderr)
-        self.assertNotIn('Starting new HTTP connection (4)', stderr)
+        self.assertIn('Retrying (Retry(total=1', stderr)
+        self.assertIn('Retrying (Retry(total=0', stderr)
```

Comment:
Nitpick: we should place the `0` statement before the `1` statement if that is the order they appear in the stderr file.

Reply:
That's actually the order they show up in